File: Rearranging array/Rearranging_array.java
class Solution {
    public static ArrayList<Integer> Rearrange(int[] arr) {
        Arrays.sort(arr);
        ArrayList<Integer> ans = new ArrayList<>();
        int l = 0, r = arr.length- 1;

        while (l <= r) {
            if (l == r) {
                ans.add(arr[l]);
            } else {
                ans.add(arr[l]);
                ans.add(arr[r]);
            }
            l++;
            r--;
        }
        return ans;
    }
}

File: Rearranging array/Rearranging_array.py
class Solution:    
    def Rearrange(self, arr):
        arr.sort()
        ans = []
        l, r = 0, len(arr) - 1

        while l <= r:
            if l == r:
                ans.append(arr[l])
            else:
                ans.append(arr[l])
                ans.append(arr[r])
            l += 1
            r -= 1

        return ans
